src/components/AboutPage/mocks/aboutData.ts

```diff
@@ -0,0 +1,95 @@
+import { GitHub, Instagram, LinkedIn } from '@mui/icons-material';
+import type { About } from '../types/About';
+
+export const aboutData: Array<About> = [
+  {
+    name: 'Togrul Kerrimov',
+    position: 'Product Owner',
+    contacts: [
+      {
+        icon: Instagram,
+        url: 'https://instagram.com/aleksandra05sh?igshid=YmMyMTA2M2Y=',
+      },
+      {
+        icon: LinkedIn,
+        url: 'https://www.linkedin.com/in/oleksandra-shevchenko-b2330122a/',
+      },
+      { icon: GitHub, url: 'https://github.com/Aleksandra05sh' },
+    ],
+    description: 'The site looks like exactly i wanted',
+    profile_image: process.env.PUBLIC_URL + '/tk_photo.jpg',
+  },
+  {
+    name: 'Vladyslav Khrychev',
+    position: 'Team mentor',
+    contacts: [
+      {
+        icon: Instagram,
+        url: 'https://instagram.com/aleksandra05sh?igshid=YmMyMTA2M2Y=',
+      },
+      {
+        icon: LinkedIn,
```

Comment:
Why Aleksandra's social is used for me and Vlad?

Reply:
It was template.
Already fixed with real data.